vendor/auth_strategies/ldap_strategy/app/forms/gobierto_admin/ldap_session_form.rb

```diff
@@ -28,9 +28,9 @@ def authentication_data_invalid?
   end
 
   def find_or_create_admin_by_ldap
-    @admin = GobiertoAdmin::Admin.regular_on_site(site).find_by(ldap_data.slice(:email)) || new_admin(ldap_data)
+    @admin = GobiertoAdmin::Admin.active.find_by(ldap_data.slice(:email)) || new_admin(ldap_data)
 
-    if admin.new_record? && admin.save
+    if admin.new_record? && admin.save || admin.regular? && admin.sites.exclude?(site)
```

Comment:
Where is admin defined as attr_accessor or reader? I can't it neither in this file nor in the parent class

Reply:
It's defined as `attr_reader :admin` in the parent class